client-src/elements/chromedash-feature-filter.js

```diff
@@ -144,6 +156,57 @@ const QUERIABLE_FIELDS = [
     type: NUM_TYPE},
   // 'browsers.chrome.ot.feedback_url': Feature.origin_trial_feedback_url,
   // 'finch_url': Feature.finch_url,
+
+  // TODO(kyleju): Use ALL_FIELDS from form-field-specs.js.
+  // Available ENUM fields.
+  {name: 'category',
+    display: 'Feature category',
+    choices: FEATURE_CATEGORIES,
+    type: ENUM_TYPE},
+  {name: 'feature_type',
+    display: 'Feature type',
+    choices: FEATURE_TYPES,
+    type: ENUM_TYPE},
+  {name: 'impl_status_chrome',
+    display: 'Implementation status',
+    choices: IMPLEMENTATION_STATUS,
+    type: ENUM_TYPE},
+  {name: 'intent_stage',
+    display: 'Spec process stage',
+    choices: INTENT_STAGES,
+    type: ENUM_TYPE},
+  {name: 'rollout_platforms',
+    display: 'Rollout platforms',
+    choices: PLATFORM_CATEGORIES,
+    type: ENUM_TYPE},
+  {name: 'standard_maturity',
+    display: 'Standard maturity',
+    choices: STANDARD_MATURITY_CHOICES,
+    type: ENUM_TYPE},
+  {name: 'security_review_status',
+    display: 'Security review status',
+    choices: REVIEW_STATUS_CHOICES,
+    type: ENUM_TYPE},
+  {name: 'privacy_review_status',
+    display: 'Privacy review status',
+    choices: REVIEW_STATUS_CHOICES,
+    type: ENUM_TYPE},
+  {name: 'tag_review_status',
+    display: 'TAG Specification Review Status',
+    choices: REVIEW_STATUS_CHOICES,
+    type: ENUM_TYPE},
+  {name: 'safari_views',
```

Comment:
These should use the dotted property names like the ones commented out on lines 123-128.   And then those commented out lines can be deleted.